src/typography/index.ts

```diff
@@ -11,3 +31,57 @@ export const Logo = styled(SVGIcon)`
   width: 50px;
   height: 50px;
 `;
+
+export const RSLogo = styled(RSLogoIcon)`
+  width: 84px;
+  height: 30px;
+`;
+
+const StyledText = css<StyledTextProps>`
```

Comment:
wow! newer used stc this way. Cool! Maybe we can write components and css declarations in different files ( daclarations with `css` in separate file, and import it in file with `styled.`  as components)?